tools/src/test/scala/caliban/tools/stitching/RemoteQuerySpec.scala

```diff
@@ -120,7 +120,7 @@ object RemoteQuerySpec extends ZIOSpecDefault {
         actual == """query{union(value:"{\"foo\": \"\\n\"}"){...on Interface{id}}}"""
       )
     },
-    test("correctly renders a query for a field of json escaped escaped quote") {
+    test("correctly renders a query for a field of json double escaped quote") {
```

Comment:
It's really hard to tell if this is intentionally doubled or accidentally, using the word `double` instead makes it clear...